---------------------------------
[2025-05-20 13:07:42][INFO][200]:La conexion [dbconn_standard] esta desabilitada en el archivo .config
---------------------------------
[2025-05-20 13:07:42][INFO][200]:Iniciando ...
---------------------------------
[2025-05-20 13:07:44][INFO][200]:Model loaded in 1.41373872756958 seconds
---------------------------------
[2025-05-20 13:07:44][INFO][200]:[INFO] Reading from rtsp://100.108.97.81:8554/cam1
---------------------------------
[2025-05-20 13:07:44][EXCEPTION][500]:PYTHON-EXCEPTION:

>>>>>>>>>>>>>>>[TRACEBACK]<<<<<<<<<<<<<<<

  File "/home/alanromero/alan/pinos-analytics-cv/src/IOT-Agent/main.py", line 10, in <module>
    ProjectDirector.go( sys.argv )
  File "/home/alanromero/alan/pinos-analytics-cv/src/IOT-Agent/System/Director/ProjectDirector.py", line 133, in go
    ProjectDirector().setFlux( argv )
  File "/home/alanromero/alan/pinos-analytics-cv/src/IOT-Agent/System/Director/ProjectDirector.py", line 113, in setFlux
    self.__play(
  File "/home/alanromero/alan/pinos-analytics-cv/src/IOT-Agent/System/Director/ProjectDirector.py", line 60, in __play
    detector = MovementDetector(
               ^^^^^^^^^^^^^^^^^
  File "/home/alanromero/alan/pinos-analytics-cv/src/IOT-Agent/System/App/MovementDetector/MovementDetector.py", line 83, in __init__
    self.recorder = RTSPRecorder(camera=camera, folder=folder, width=width, height=height, verbose=verbose, visualize=visualize)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/alanromero/alan/pinos-analytics-cv/src/IOT-Agent/System/App/RTSPRecorder/RTSPRecorder.py", line 30, in __init__
    self.frame = np.zeros((height, width, 3), dtype=np.uint8)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

>>>>>>>>>>>>>>[ERROR_DESC]<<<<<<<<<<<<<<<

  [TypeError]:'NoneType' object cannot be interpreted as an integer

>>>>>>>>>>>>>>>>>>>><<<<<<<<<<<<<<<<<<<<<

